### PR TITLE
Fix nav unification to load the correct state module init

### PR DESCRIPTION
**Pull request by getdave**
#### Changes proposed in this Pull Request

Fixes and error with the Nav Unification modularized state which was loading the `inline-help/init` file rather than the `admin-menu/init` file. I have no idea how this happened or how this has ever worked. This PR resolves things to use the correct init.

#### Testing instructions

* Apply D49005-code
* Apply blog sticker
* Run calypso
* Visit http://calypso.localhost:3000/?flags=nav-unification.
* You should see left hand Nav rendered so match API output as generated by D49005-code.

Basically nothing should change. It should be "as is".

### CODE REVIEW

**Review by cpapazoglou — APPROVED** [review]
Interesting! LGTM!

**Comment by mreishus** [comment]
Oops - I must have copy and pasted this without updating it.  Nice catch! :+1:
